server/datastore/mysql/maintained_apps_test.go

```diff
@@ -58,7 +58,7 @@ func testUpsertMaintainedApps(t *testing.T, ds *Datastore) {
 		Token:        "figma",
 		InstallerURL: "https://desktop.figma.com/mac-arm/Figma-999.9.9.zip",
 		Version:      "999.9.9",
-		Platform:     fleet.MacOSPlatform,
```

Comment:
Why the move away from the constant?

Reply:
The constant is a string type specifically for Apple platforms